collect_app/src/main/java/org/odk/collect/android/analytics/AnalyticsEvents.java

```diff
@@ -72,6 +72,12 @@ private AnalyticsEvents() {
      */
     public static final String AUDIO_QUESTION = "Prompt";
 
+    /**
+     * Track form definitions with url widget. The action should be the widget name and
+     * the label should be a hash of the form definition.
+     */
+    public static final String URL_WIDGET = "Prompt";
```

Comment:
This doesn't seem right as it's the same as the `AUDIO_QUESTION`. I think the `AUDIO_QUESTION` naming and description is just wrong here and probably should be `PROMPT` and really be used to just track any kind of question/widget.